Examples/Swift/CustomViewController.swift

```diff
@@ -73,17 +74,6 @@ class CustomViewController: UIViewController, MGLMapViewDelegate, AVSpeechSynthe
         addRouteToMap()
     }
 
-    // When an instruction should be given
-    @objc func shouldSpeak(_ notification: NSNotification) {
-        let routeProgress = notification.userInfo![MBRouteControllerDidPassSpokenInstructionPointRouteProgressKey] as! RouteProgress
-        
-        guard let text = routeProgress.currentLegProgress.currentStepProgress.currentSpokenInstruction?.text else { return }
-
-        let utterance = AVSpeechUtterance(string: text)
-        speechSynth.delegate = self
-        speechSynth.speak(utterance)
```

Comment:
The intent of this code was to demonstrate using AVSpeechSynthesizer as opposed to Polly. `speechSynth` (still present in this PR) is an instance of AVSpeechSynthesizer.